### PR TITLE
fix: inline table editing does not retain validation error after cancel

### PR DESCRIPTION
**Pull request by tracy-french**
# Description

The change resolves https://github.com/awslabs/iot-application/issues/103.

## Legal

This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: apps/client/src/routes/dashboards/dashboards-index/dashboards-index-page.tsx [hunk 1]
```diff
@@ -58,6 +59,7 @@ export function DashboardsIndexPage() {
             [column.id]: getValues(column.id),
           });
         }}
+        onEditCancel={() => resetForm()}
```
Comment (boweihan) [inline apps/client/src/routes/dashboards/dashboards-index/dashboards-index-page.tsx:62]: love one liners.

**Review by boweihan — DISMISSED** [review]
LGTM. Nit: would be nice to have a one sentence description about what the fix is in the PR description, though this one is pretty self evident.

**Comment by tracy-french** [comment]
> LGTM. Nit: would be nice to have a one sentence description about what the fix is in the PR description, though this one is pretty self evident.

lol yes, you're right.

**Comment by tracy-french** [comment]
Looks like the tests are timing out because of the waitFor's. Thanks React!

**Review by boweihan — APPROVED** [review]

**Pull request merged**